File: composeApp/src/androidMain/kotlin/com/app/starleet/states/LactateState.kt
package com.app.starleet.states

import com.app.starleet.roomdb.LactateScanEntity

data class LactateState(
    val current: LactateScanEntity? = null,
    val history: List<LactateScanEntity> = emptyList(),
    val trend: List<LactateScanEntity> = emptyList(),
    val isLoading: Boolean = false,
    val error: String? = null,
    val selectedDate: Long? = null,
)

File: iosApp/iosApp/State/LactateState.swift
//
//  LactateState.swift
//  iosApp
//
//  Created by KBS on 3/3/26.
//
import AVFoundation

struct LactateState {
    var current: LactateScan?
    var history: [LactateScan] = []
    var trend: [LactateScan] = []
    var isLoading: Bool = false
    var error: String?
    var selectedDate: Date?
}
